packages/sync-actions/test/product-sync-prices.spec.js

```diff
@@ -217,6 +217,56 @@ describe('Actions', () => {
             validFrom: dateNow,
             validUntil: twoWeeksFromNow,
           },
+          {
+            // set price custom type
+            id: '888',
+            value: { currencyCode: 'GBP', centAmount: 1000 },
+            country: 'UK',
+            validFrom: dateNow,
+            validUntil: twoWeeksFromNow,
+          },
+          {
+            // set price custom type and field
+            id: '999',
+            value: { currencyCode: 'GBP', centAmount: 1000 },
+            country: 'UK',
+            validFrom: dateNow,
+            validUntil: twoWeeksFromNow,
+          },
+          {
+            // change price custom field
+            id: '1010',
+            value: { currencyCode: 'GBP', centAmount: 1000 },
+            country: 'UK',
+            validFrom: dateNow,
+            validUntil: twoWeeksFromNow,
+            custom: {
+              type: {
+                typeId: 'type',
+                id: '5678',
+              },
+              fields: {
+                source: 'shop',
+              },
+            },
+          },
+          // {
+          //   // remove price custom field
+          //   id: '1111',
+          //   value: { currencyCode: 'GBP', centAmount: 1000 },
+          //   country: 'UK',
+          //   validFrom: dateNow,
+          //   validUntil: twoWeeksFromNow,
+          //   custom: {
+          //     type: {
+          //       typeId: 'type',
+          //       id: 'customType1',
+          //     },
+          //     fields: {
+          //       source: "shop",
+          //     },
+          //   },
+          // },
```

Comment:
I didnt add this test cause sth weird happened, when i send this actions to platform 
`
{
  "version": 15,
  "actions": [
{
    "action": "setProductPriceCustomType",
    "priceId": "f28c248b-91e9-4f33-aeda-7237ecdabb60"
  }
  ]
}
`
This deletes the price object, i will give it more time to make sure its the platform